### DISCUSSION TITLE
Bad redirect in github.com/events

### DISCUSSION THREAD

**Discussion by JhonnyLusonode** [community/community#19368] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everyone

If you are having trouble opening the Kansas City Developer Conference 2022 link in:
https://github.com/events 

Sponsored by GitHub
[Kansas City Developer Conference 2022](hhttps://www.kcdc.info/)  <--- there is 2 "h" on the https
Aug 8 - 10, 2022 • Kansas City, MO

Just remove one of them and you will be able to join the website.
https://www.kcdc.info/

Thank you :D

**Comment by melvincwng** [comment, marked as answer]
@JhonnyLusonode, seems like there is a broken link in that webpage.

Kindly report it to [Github Bug Report](https://support.github.com/contact/bug-report) over here.

Hope this clarifies.

**Reply by JhonnyLusonode** [reply]
Actually didn´t know where to report it.
Thank you! :D

**Reply by melvincwng** [reply]
@JhonnyLusonode, pleasure to help ;)
